src/exponential-backoff.js

```diff
@@ -17,10 +17,10 @@
 
 /**
  * @param {opt_base} opt_base Exponential base. Defaults to 2.
- * @return {function(function())} Function that when invoked will
+ * @return {function(function(): number)} Function that when invoked will
  *     call the passed in function. On every invocation the next
  *     invocation of the passed in function will be exponentially
- *     later.
+ *     later. Returned function returns timeout id.
```

Comment:
@dvoytenko do you mean the actual function? this is used in error.js for global backoff